apps/web/components/sections/about-us/why-is-different.tsx

```
@@ -0,0 +1,52 @@
+        className="text-center max-w-2xl"
+      >
+        <h2 className="text-4xl font-bold">Why Is Different</h2>
+        <p className="text-lg text-gray-600 mt-2">
+          Discover what makes this platform unique and how it revolutionizes crowdfunding.
+        </p>
+      </motion.div>
+
+      <div className="relative flex justify-center gap-6 w-full px-6">
+        {whyIsDifferent.map((feature, index) => (
+          <motion.div
+            key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)